plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzer.py

```diff
@@ -345,31 +345,35 @@ def create_analyzer(parsed_args, full_extracted_logs_list,
         pdf_header = (
             f"Dump analysis for {os.path.basename(args.location)}, hours={args.hours}"
         )
-        FABRIC_INFO = str(ibdiagnet_analyzer.get_fabric_size() \
+
+        used_ufm_version = console_log_analyzer.ufm_versions
+        text = f"Used ufm version in console log {used_ufm_version}"
+        FABRIC_INFO = "fabric info:" + os.linesep + str(ibdiagnet_analyzer.get_fabric_size() \
                         if ibdiagnet_analyzer else "No Fabric Info found")
 
         LINK_FLAPPING = str(links_flapping_analyzer.get_link_flapping_last_week() \
                             if links_flapping_analyzer else "No link flapping info")
         # PDF creator gets all the images and to add to the report
-        TEXT = FABRIC_INFO + os.linesep + "Link Flapping:" + os.linesep + LINK_FLAPPING
+        TEXT = text + os.linesep + FABRIC_INFO + os.linesep + \
+        "Link Flapping:" + os.linesep + LINK_FLAPPING
         pdf = PDFCreator(pdf_path, pdf_header, png_images, TEXT)
         pdf.created_pdf()
         # Generated a report that can be located in the destination
         log.LOGGER.info("Analysis is done, please see the following outputs:")
         for image, title in images_and_title_to_present:
             log.LOGGER.info(f"{title}: {image}")
         log.LOGGER.info(f"Summary PDF was created! you can open here at {pdf_path}")
+        if args.interactive:

```

Comment:
is it related to this PR?

Reply:
No, a small improvement....
Has no effect on the final result